packages/babel-plugin-transform-runtime/test/fixtures/runtime-corejs2/full/output.mjs

```diff
@@ -1,6 +1,6 @@
 import _regeneratorRuntime from "@babel/runtime-corejs2/helpers/regeneratorRuntime";
 var _marked = /*#__PURE__*/_regeneratorRuntime().mark(giveWord);
-import _Symbol from "@babel/runtime-corejs2/core-js/symbol";
+import _Symbol from "@babel/runtime-corejs2/core-js/symbol/index";
```

Comment:
There seem to be some issues here.
![image](https://github.com/babel/babel/assets/30521560/bd34e4d6-cc92-4968-aa76-2ad0e72f41b1)
![image](https://github.com/babel/babel/assets/30521560/fcfc2dc2-18bf-4ad1-9af9-53eebb5bba0d)

Reply:
Ushhh thanks for catching this